apps.yml

```diff
@@ -3638,7 +3638,7 @@ apps:
     logo: auth0.png
     name: settings.dev.mozaws.net
```

Comment:
ugh.  I click approve right as I see something.
Should the name line change to match the URL?